### PR TITLE
More markups to the user-facing sign-up copy

### PR DESCRIPTION
**Pull request by alexwlchan**
For https://github.com/wellcomecollection/wellcomecollection.org/issues/8160 and https://github.com/wellcomecollection/wellcomecollection.org/issues/8178

### CODE REVIEW

File: identity/webapp/copy.tsx [hunk 1]
```diff
@@ -32,9 +32,20 @@ export const ValidatedSuccessText: FC<ValidatedSuccessTextProps> = ({
...
-          If you want to access subscription databases, e-journals and e-books,
-          you need to bring a form of personal identification (ID) and proof of
-          address to the admissions desk in the library.
+          To complete your membership and access subscription databases,
+          e-journals and e-books, you’ll need to bring a form of photo
+          identification (ID) and proof of your address to our admissions desk
+          when you visit.
+        </p>
+        <p>
+          <strong>Reminder:</strong> you will need to email a form of personal
```
Comment (melanierogan) [inline identity/webapp/copy.tsx:41]: Were we trying to keep references of emailing personal id out of text?

**Review by melanierogan — COMMENTED** [review]
Just one point for clarification. I realise we need to ensure we explain what a user needs to do to with ID, but I had thought we were trying to keep reference to emailing id to minimum; Having said this I don't know how else we are going to explain this process.

File: identity/webapp/copy.tsx (hunk 1 above)
Comment (alexwlchan) [reply identity/webapp/copy.tsx:41]: Argh, thanks for catching

Comment (alexwlchan) [reply identity/webapp/copy.tsx]: Fixed now.

**Pull request merged**